src/components/Insight/InsightsTrends.js

```diff
@@ -17,6 +17,8 @@ const InsightsTrends = ({ allInsightsByTag, ...props }) => {
   )
 }
 
+export const oneDayTimestamp = 1000 * 60 * 60 * 24
```

Comment:
Use [this const](https://github.com/santiment/app/blob/9c0007579f7004e828ad668ef8dc78c24b9ba3d8/src/utils/dates.js#L4) from a `utils/dates`